Linear_cell_complex/doc/Linear_cell_complex/Concepts/LinearCellComplexTraits.h

```diff
@@ -52,7 +52,7 @@ Functor that provides
 \link LinearCellComplexTraits::Point `Point`\endlink `operator() (const` \link Point `Point`\endlink`& p, const` \link LinearCellComplexTraits::Vector `Vector`\endlink`& v)`,
 which constructs the translation of point `p` by vector `v`, and
 \link LinearCellComplexTraits::Point `Point`\endlink `operator() (const CGAL::Origin&, const` \link LinearCellComplexTraits::Vector `Vector`\endlink`& v)`,
-which constructs the translation of a point at the origin by vector `v` (used in \link CGAL::LinearCellComplex::barycenter `LinearCellComplex::barycenter`\endlink).
```

Comment:
@gdamiand what is the expected function you want to link to?

Reply:
ping @gdamiand

Reply:
This one https://doc.cgal.org/latest/Linear_cell_complex/classLinearCellComplex.html#a0689ef15676c7d3082abcdbe64c364af